README.md

```diff
@@ -13,6 +13,14 @@ Restricting what's in your runtime container to precisely what's necessary for y
 and other tech giants that have used containers in production for many years.
 It improves the signal to noise of scanners (e.g. CVE) and reduces the burden of establishing provenance to just what you need.
 
+Distroless images are _very small_.
```

Comment:
This only considers non-language image. The Distroelss Java 11 is 200MB, which is not much different from other images. (The Java 8 image, which is defunct, is of size 130MB while `openjdk:8-jre-alpine1 is 84.9MB.)

Reply:
It's a fair point. Is it maybe better to say "Distroless images *aim to be* very small"?

It looks like gcr.io/distroless/java-debian10:11 is quite a bit smaller than openjdk:11 though (71 MB vs 314 MB), so I think we could still make a blanket statement that distroless images are smaller than their distroful counterparts.

Reply:
`openjdk:11` is a full JDK. It's not a fair comparison.
```
gcr.io/distroless/java-debian10   11             8775134e5ec2   51 years ago    200MB
adoptopenjdk                      11-jre         ca04986fae0a   12 days ago     244MB
openjdk                           11             e782567c0965   4 days ago      648MB
```

Reply:
And alpine is much smaller.
```
adoptopenjdk/openjdk11            alpine-jre              7af79a5acb07   22 hours ago    148MB
```

Reply:
Fair enough. Is there some way we can document the relative smallness of distroless images (even though there might exist smaller alternatives) clearly? If you think the comparison isn't worth mentioning that's fine too, mainly I don't want to have the README take a long tangent covering the relative sizes of all possible alternatives. I just wanted to point out that distroless/static is smaller than the GitHub repo's front page 😄

Reply:
I don't have a strong opinion, so don't block on me. I'm not a maintainer after all. Do what you think is appropriate.